package.json

```diff
@@ -12,12 +12,18 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "mocha"
+    "build": "babel es --out-dir lib",
+    "test": "npm run build && mocha",
+    "prepublish": "npm test"
   },
   "files": [
     "index.js",
     "ponyfill.js",
-    "index.d.ts"
+    "index.d.ts",
+    "es/index.js",
```

Comment:
I think you’d need to add `"jsnext:main": "es/index.js"` for Rollup to discover the ES build. The `es` folder name is just a convention I use.